rap_connector_agora/models/agora_connector.py

```diff
@@ -653,7 +674,7 @@ def product_data(self, product, is_new, connection):
                     "IsSoldByWeight": product.is_sold_by_weight,
                     "AskForPreparationNotes": product.ask_preparation_notes,
                     "AskForAddins": product.ask_for_addings,
-                    "PrintWhenPriceIsZero": product.print_zero,
+                    "PrintWhenPriceIsZero": not product.print_zero,
```

Comment:
Esto es un diccionario?? Puedes ponerle not delante de esa asignación?

Reply:
esto lo uqe hace es negar el valor boolean que tiene este campo, porque hemos detectado que agora lo muestra invertido... es decir si en la api me viene True ese campo en Agora se ve False, no comprendo por que pero lo tienen asi